yt/data_objects/static_output.py

```diff
@@ -119,10 +120,7 @@ def __init__(self, display_array=False):
         # We can assume that ipywidgets will not be *added* to the system
         # during the course of execution, and if it is, we will not wrap the
         # array.
-        if display_array and find_spec("ipywidgets") is not None:
-            self.display_array = True
-        else:
-            self.display_array = False
+        self.display_array = display_array and IPYWIDGETS_ENABLED
```

Comment:
If I am remembering correctly, some of this was added to avoid calling `find_spec` too many times, which led to a pretty bonkers number of import checks at runtime.  Can you assure that this does not re-introduce that performance hit?

Reply:
Yup. `IPYWIDGETS_ENABLED` is only evaled once.